providers/implementations/keymgmt/ec_kmgmt.c

```diff
@@ -549,6 +565,18 @@ int ec_set_params(void *key, const OSSL_PARAM params[])
     if (p != NULL && !ec_set_param_ecdh_cofactor_mode(eck, p))
         return 0;
 
+    p = OSSL_PARAM_locate_const(params, OSSL_PKEY_PARAM_TLS_ENCODED_PT);
+    if (p != NULL) {
```

Comment:
please check that the data type is correct

Reply:
Fixed